### PR TITLE
Update anaconda to 5.2.0

### PR DESCRIPTION
**Pull request by brianbrownton**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/anaconda.rb [hunk 1]
```diff
@@ -1,11 +1,10 @@
...
-  sha256 'be705b3c3a0ca29ee32ce7658890bb5edb32a9eadedc09dec3d7e3cfbfd23cb7'
+  version '5.2.0'
+  sha256 'c8089121dc89ffe8f9a0c01205bab75a112821a13d413152d6690f5eef094afa'
 
-  # repo.continuum.io/archive was verified as official when first introduced to the cask
-  url "https://repo.continuum.io/archive/Anaconda3-#{version}-MacOSX-x86_64.sh"
+  url "https://repo.anaconda.com/archive/Anaconda3-#{version}-MacOSX-x86_64.sh"
   name 'Continuum Analytics Anaconda'
-  homepage 'https://www.anaconda.com/what-is-anaconda/'
+  homepage 'https://www.anaconda.com/'
```
Comment (adidalal) [inline Casks/anaconda.rb]: Why the change? Main page doesn't mention specifics about the anaconda package

Comment (brianmorton) [reply Casks/anaconda.rb]: @adidalal There is a big "download" button in the top right of the page, and per the docs ( https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/url.md )

> Where URL_SECTION is the smallest possible portion of the URL that uniquely identifies the app or vendor.

I thought the previous URL was not the smallest possible portion.
